pages/opening-days-some.js

```diff
@@ -20,81 +26,85 @@ const OpeningSomeDays = (props) => (
         validatorErrors={props.validatorErrors}
         onHandleErrorClick={OnHandleErrorClick}
       />
-      <Heading as="h1" size="LARGE">
-        {props.t("Opening days")}
-      </Heading>
-      <Paragraph>
-        {props.t(
-          "What days will this establishment be producing or serving food?"
-        )}
-      </Paragraph>
-      <Paragraph>{props.t("Select all that apply")}</Paragraph>
-      <ContentItem.B_30_15>
-        <MultiChoice
-          label=""
-          meta={{
-            touched: true,
-            error: props.t(props.validatorErrors.opening_days_some)
-          }}
-        >
-          <Checkbox
-            name="opening_day_monday"
-            id="opening_day_monday"
-            value="Monday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_monday}
-          >
-            {props.t("Monday")}
-          </Checkbox>
-          <Checkbox
-            name="opening_day_tuesday"
-            id="opening_day_tuesday"
-            value="Tuesday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_tuesday}
-          >
-            {props.t("Tuesday")}
-          </Checkbox>
-          <Checkbox
-            name="opening_day_wednesday"
-            id="opening_day_wednesday"
-            value="Wednesday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_wednesday}
-          >
-            {props.t("Wednesday")}
-          </Checkbox>
-          <Checkbox
-            name="opening_day_thursday"
-            id="opening_day_thursday"
-            value="Thursday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_thursday}
-          >
-            {props.t("Thursday")}
-          </Checkbox>
-          <Checkbox
-            name="opening_day_friday"
-            id="opening_day_friday"
-            value="Friday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_friday}
-          >
-            {props.t("Friday")}
-          </Checkbox>
-          <Checkbox
-            name="opening_day_saturday"
-            id="opening_day_saturday"
-            value="Saturday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_saturday}
-          >
-            {props.t("Saturday")}
-          </Checkbox>
-          <Checkbox
-            name="opening_day_sunday"
-            id="opening_day_sunday"
-            value="Sunday"
-            defaultChecked={props.cumulativeFullAnswers.opening_day_sunday}
+      <Fieldset>
+        <Fieldset.Legend>
```

Comment:
Instead of having a Heading within the Legend, you should be able to add the appropriate attributes to the Legend element:

<Fieldset.Legend
            size="LARGE"
            isPageHeading
            style={{ marginBottom: "30px" }}
          >
           {props.t("Opening days")}
          </Fieldset.Legend>